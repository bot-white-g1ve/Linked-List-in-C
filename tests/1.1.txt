NEW 1
a
NEW
NEW0
NEW 0
NEW 5
 6e-1
 4 
 4 4

   
NEW 2
{
  1}
NEW 4
a√ü