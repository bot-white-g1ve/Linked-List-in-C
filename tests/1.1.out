List 0: a
INVALID COMMAND: NEW
INVALID COMMAND: INPUT
List 1: 
List 2: 0.60 -> 4 ->  4 4 ->  ->    
INVALID COMMAND: NEW
